osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -22,10 +21,9 @@ namespace osu.Game.Overlays.Profile.Sections.Ranks
     public class DrawableProfileScore : CompositeDrawable
     {
         private const int height = 40;
-        private const int performance_width = 80;
+        private const int performance_width = 100;
 
         private const float performance_background_shear = 0.45f;
-        private static readonly float performance_background_width = performance_width + (height / 4f * MathF.Tan(performance_background_shear));
```

Comment:
This shouldn't be removed. See #7710.

Reply:
It didn't need to be there in the first place, if we're going for absolute sizing parity with web.

| this PR | web |
| :-: | :-: |
| ![image](https://user-images.githubusercontent.com/20418176/74600830-2b99b980-5097-11ea-87a9-111acc9e4138.png) | ![image](https://user-images.githubusercontent.com/20418176/74600800-d8276b80-5096-11ea-967d-ee4a5e7ff120.png) |

Note the consistent 100 width in both places. The previous solution wasn't really necessary and I should have paid better attention during review. Going forward I will be checking element dimensions against web very strictly.